Readme.md

```diff
@@ -0,0 +1,171 @@
+
+Then, the required inputs and secrets should be included/adapted based on the needs of the repository. 
+
+### Triggering a workflow
+Workflow triggers are events that cause a workflow to run. All the Continuous Integration workflows can be manually run from the Actions tab of your repository. 
+
+![Action tab](assets/images/learn-github-actions-repository.png)
+
+Write access to the repository is required to perform these steps.
+
+1. In the left sidebar, click the workflow you want to run.
+
+    ![Action select](assets/images/actions-select-workflow.png)
+
+2. Above the list of workflow runs, select Run workflow.
+
+    ![Action dispatch](assets/images/actions-workflow-dispatch.png)
+
+3. Use the Branch dropdown to select the workflow's branch. Click Run workflow.
+
+    ![Action run](assets/images/actions-manually-run-workflow.png)
+
+### Viewing a workflow
+
+After a workflow run has started, you can see a visualization graph of the run's progress and view each step's activity on GitHub.
+
+1. In the left sidebar, click the workflow you want to see.
+
+    ![Workflow](assets/images/learn-github-actions-workflow.png)
+
+2. Under "Workflow runs", click the name of the run you want to see.
+
+    ![Workflow run](assets/images/learn-github-actions-run.png)
+
+3. Under Jobs or in the visualization graph, click the job you want to see.
+
+    ![Job result](assets/images/overview-actions-result-navigate.png)
+
+4. View the results of each step.
+
+    ![Step result](assets/images/overview-actions-result-updated-2.png)
+
+### Updating environment variables (if needed)
+The environment variables are passed to the caller workflows as either inputs or secrets. 
+Whenever you want to add a new environment variable to your repository and pass it as a secret, you should update your workflow as well. 
+
+For this purpose, you need to perform the following steps. 
+#### 1. Update inputs and secrets in a reusable workflow definition
+```
+on:
+  workflow_call:
+    inputs:
+      input-variable:
+        required: true
+        type: string
+    secrets:
+      secret-variable:
+        required: true
+```
+
+#### 2. Reference the input or secret that you defined in the on key in the previous step.
+
+```
+jobs:
+  reusable_workflow_job:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ./.github/workflows/...
+        with:
+          input-variable: ${{ inputs.input-variable }}
+          secret-variable: ${{ secrets.secret-variable }}
+```
+
+#### 3. Include the secret in the caller workflow. 
+- Use the `with` keyword to pass named inputs.
+- Use the `secrets` keyword to pass named secrets. 
+````
+jobs:
+  call-workflow-passing-data:
+    uses: octo-org/example-repo/.github/workflows/reusable-workflow.yml@main
```

Comment:
`uses` our workflow?

Reply:
yes. I clarifies this in the new version